lib/player/stream_video_source.js

```diff
@@ -1135,6 +1141,18 @@ shaka.player.StreamVideoSource.prototype.startStreams_ = function(
   this.enableTextTrack(this.subsNeeded_);
 };
 
+/**
+ * Sets the currentTime of the video element.  This is deferred if metadata
+ * has not been loaded yet.
+ *
+ * @param {number} streamStartTime
+ * @private
+ */
+shaka.player.StreamVideoSource.prototype.setInitialCurrentTime = function(
```

Comment:
Run the compiler & linter with build/all.sh.  It will complain that this doesn't end in an underscore, which is the style we follow for private functions.